### PR TITLE
Update microsoft-teams-rooms-configure-servicenow.md

### PR DESCRIPTION
**Pull request by lovelight68**
Adding notification that SNOW API is not available in GCC

### CODE REVIEW

File: Teams/rooms/microsoft-teams-rooms-configure-servicenow.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ This article describes the prerequisites and steps to configure your ServiceNow
 ### Teams Rooms prerequisites
 
 - You must have an assigned Service Administrator role. For more information, see [Role-based access control with Microsoft Teams Rooms Pro Management](rooms-pro-rbac.md).
+- Please note that the ServiceNow API is not currently available for GCC customers. We are reviewing solutions to enable this in government clouds. For integration with your ITSM workflow, it is recommended that customers consider using email notifications and your ITSM email ingest features.
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-configure-servicenow.md:35]:
```suggestion
- Please note that the ServiceNow API is not currently available for GCC customers. We are reviewing solutions to enable this in government clouds. For integration with your ITSM workflow, we recommend that you consider using email notifications and your ITSM email ingest features.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lovelight68, kindly commit the suggested change. Many thanks.

**Comment by lovelight68** [comment]
This is fine. Please feel free to publish.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@altsou 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**